department-of-veterans-affairs/va.gov-cms
# VAGOV-6366 add link reporting

Testing Steps:
Vist existing content with diverse links validate that node report shows  and check for appropriateness of the report:
- [x] (lots of links) /pittsburgh-health-care/locations/belmont-county-va-clinic
- [x] (Lots of links, odd lnks to blocks) /health-care
- [x] (link to non drupal page) /health-care/about-va-health-benefits/vision-care/blind-low-vision-rehab-services
- [x] (911 link fatal error) /pittsburgh-health-care/operating-status

Create a node of any content type that has fields that support links.

- [x] insert link to published internal node
- [x] insert link to unpublished node
- [x] insert ink to valid external site (https://amazon.com)
- [x] insert link to non-existent site (https://weatherunderBound.com)
- [x]  save the node.  Validate that node report shows up showing one broken link and one unpublished link.
- [x] Edit the node and validate that the node report shows on node edit.
- [x] add a newlink to a broken external link (https://no-site-to=-e-found-here.gov)
- [x] save the node and validate that now there are two broken links. and 1 unpublished warning link
